Aditi/aditi_first_assignment/style.css

```diff
@@ -0,0 +1,102 @@
+ body{
+	 width:100%;
+	 height:100vh;
+	 background-image:url("bg.jpg.jpg");
+	 background-size:cover;
+	 
+	 }
+	 
+	 section.header-section{
+		 border-top:0px;
+	  background-color:white;
+	  min-height:50px;	
+	   padding-left:100px;
+	  padding-right:100px;
```

Comment:
remove this padding and add it to body tag.
Also add float: left to your section here.